bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -2,6 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% set_lang_files "foundation/base-resp" %}
```

Comment:
@flodolo do you have a preference here? Maybe a different name? Didn’t want to add the sidebar strings to main.lang since those are opt-in pages